README.md

```diff
@@ -65,7 +65,7 @@ builder.build({
 
 Creates a new `Builder` instance.
 
-It caches build results to only rebuild a theme when related files have been changed.  
```

Comment:
Did you remove those by intension? Actually I've put the two spaces to have a line-break.

Reply:
Oh sorry, this probably was changed on save, I didn't notice as I only checked the source files when creating the PR.